src/routes/safe/components/Transactions/helpers/TxParametersDetail/index.tsx

```diff
@@ -87,7 +84,7 @@ const TxParameter = ({ name, value, ...rest }: TxParameterProps): ReactElement |
   return (
     <TxParameterWrapper>
       {getEl(name)}
-      {getEl(value)}
+      {value == null || value === '' ? <Skeleton animation="wave" width="30px" /> : getEl(value)}
```

Comment:
This is used in multiple places in the Safe. Have you checked them all? If the parameter has no value then this will appear to load endlessly.

Reply:
Good catch, I reverted the change and used a normal text component with the Skeleton for `safeNonce`.